pages/mint-calendar/[launchpad_id]/project/[project_id].js

```diff
@@ -37,8 +83,41 @@ const ProjectPage = ({ project }) => {
 		cover: parseImgUrl(project.banner_image, null, { useOriginal: true }),
 	}
 
+	useEffect(() => {
+		if (!currentUser) {
+			setShowRemindMe(false)
+		} else {
+			setShowRemindMe(true)
+			const pref =
+				window.localStorage.getItem(`${router.query.launchpad_id}:${router.query.project_id}`) ||
+				null
+			if (pref == null && project.reminder && project.reminder.includes(currentUser)) {
+				setRemindMe(true)
+			} else if (pref != null) {
+				setRemindMe(pref === 'true')
+			}
+			if (!checkEmailShowed) {
+				setShowFillEmail(!currentEmail)
+			}
+		}
+	})
+
```

Comment:
ini `useEffect`nya ga dikasih dependecy props om? soalnya ini bakal ke-run in every render.
[Info](https://beta.reactjs.org/learn/synchronizing-with-effects#step-2-specify-the-effect-dependencies)